component/disassembler/disassembler.js

```diff
@@ -435,4 +435,20 @@ function Disassembler() {
     this.disassemble = disassembleMemory;
     this.loadSymbols = loadSymbols;
     this.labelAddress = labelAddress;
-}
\ No newline at end of file
+}
+
+if (typeof module === 'object' && typeof module.exports === 'object') {
+    // 当前环境是 NodeJS 规范环境
```

Comment:
```suggestion
    /* We are currently in NodeJS environment, import dependencies */
```
Please use English comments in the source code

Reply:
This habit is not good😂, I will correct it.

Reply:
Deleted.